service_container/calls.rst

```diff
@@ -71,3 +71,74 @@ To configure the container to call the ``setLogger`` method, use the ``calls`` k
 
         $container->register(MessageGenerator::class)
             ->addMethodCall('setLogger', [new Reference('logger')]);
+
```

Comment:
Maybe adding some kind of headline here to have a better structured page? 🤔

Reply:
Can you explain what you're thinking about? I'm not sure to understand it clearly 😄 🤔